staging/src/k8s.io/apiserver/pkg/storage/selection_predicate.go

```diff
@@ -71,13 +71,15 @@ func (f AttrFunc) WithFieldMutation(fieldMutator FieldMutationFunc) AttrFunc {
 
 // SelectionPredicate is used to represent the way to select objects from api storage.
 type SelectionPredicate struct {
-	Label               labels.Selector
-	Field               fields.Selector
-	GetAttrs            AttrFunc
-	IndexLabels         []string
-	IndexFields         []string
-	Limit               int64
-	Continue            string
+	Label       labels.Selector
+	Field       fields.Selector
+	GetAttrs    AttrFunc
+	IndexLabels []string
+	IndexFields []string
+	Limit       int64
+	Continue    string
+	// AllowWatchBookmarks when set indicates that the client
+	// want to receive bookmark events.
```

Comment:
I would say that this particular field is self-explaining, so I would remove this comment.

Reply:
Once you understand the meaning of that field, of course it is :)

I'm actually considering making it even more explicit, like this:
```
// AllowWatchBookmarks, when set, indicates that the end-client
// wants to receive optional bookmark events.
//
// As of today, this will be set when the 'allowWatchBookmarks'
// query parameter is set.
```

In the end, having more documentation won't harm, right? Thoughts?